deepchem/feat/tests/test_mxmnet_featurizer.py

```diff
@@ -0,0 +1,96 @@
+"""
+Test for MXMNet Featurizer class.
+"""
+
+from deepchem.feat.molecule_featurizers.mxmnet_featurizer import MXMNetFeaturizer
+import numpy as np
+
+edge_index_original_ordering = {
+    "C1=CC=NC=C1":
+        np.asarray([[0, 1, 1, 5, 5, 2, 2, 4, 4, 3, 3, 0],
+                    [1, 0, 5, 1, 2, 5, 4, 2, 3, 4, 0, 3]]),
+    "CC(=O)C":
+        np.asarray([[0, 3, 3, 2, 3, 1], [3, 0, 2, 3, 1, 3]]),
+    "C":
+        np.empty((2, 0), dtype=int)
+}
+
+node_features_original = {
+    "C1=CC=NC=C1": np.asarray([[1.], [1.], [2.], [1.], [1.], [1.]]),
+    "CC(=O)C": np.asarray([[1.], [1.], [3.], [1.]]),
+    "C": np.asarray([[1]])
+}
+
+# Set up tests.
```

Comment:
In deepchem, we generally use `unittest` class for the unit tests. Try to use that library here as well.

Reply:
https://github.com/deepchem/deepchem/blob/master/deepchem/feat/tests/test_mol_graph_conv_featurizer.py: example for the unit tests

Reply:
Sure, done.